### ISSUE TITLE
Get named destinations to ignore zoom in destination.

### ISSUE THREAD

**Issue by farling42** [farling42/fvtt-pdf-pager#60]
There is a pdfjs flag ignoreDestinationZoom which will ignore the zoom level in named destinations.

We should set this flag to true if an explicit zoom has been set in the module settings.

Unfortunately this flag needs to be set BEFORE the PdfLinkService is created, but AFTER the viewer.html has got far enough to create the preferences object.

**Comment by farling42** [comment]
In pdfjs, app.js creates a single PDFLinkService object, and passes ignoreDestinationZoom from AppOptions.get("ignoreDestinationZoom")

But the problem is setting the value in AppOptions between AppOptions being created and the PDFLinkService being created.

**Comment by farling42** [comment]
We need to call `AppOptions.set("ignoreDestinationZoom", true)` very early.

**Comment by farling42** [comment]
Some notes:
see 
```js
document.addEventListener("webviewerloaded", function() {
  PDFViewerApplication.initializedPromise.then(function() {
    // The viewer has now been initialized.
  })
});
```
from https://github.com/mozilla/pdf.js/wiki/Third-party-viewer-usage

**Comment by farling42** [comment]
Note that `updatePdfView` could probably just call `linkService.setHash(anchor)`

**Comment by farling42** [comment]
Note that AppOptions might be available at `event.target.contentWindow.PDFViewerApplicationOptions`

**Comment by farling42** [comment]
Implemented in 0.40.0
